_data/projects/hyperviz.yml

```diff
@@ -0,0 +1,11 @@
+name: HyperViz
+desc: Analysing hyper-parameters made easy.
+site: https://github.com/F1nnM/HyperViz
+tags:
+  - python
+  - streamlit
+  - web
+  - machine-learning
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/F1nnM/HyperViz/issues
```

Comment:
@F1nnM Please use the link in the form of https://github.com/F1nnM/HyperViz/labels/good%20first%20issue.